src/helpers/utils.js

```diff
@@ -5,33 +5,45 @@ const { repoMapping } = require("../constants");
  * if `except` is provided, it should be a string of characters to not escape
  * https://core.telegram.org/bots/api#markdownv2-style
  */
-const escapeMarkdown = (str, except = "") => {
+const escapeMarkdown = (str, except = "") =>
+{
   const all = "_*[]()~`>#+-=|{}.!\\".split("").filter((c) => !except.includes(c));
   const regExSpecial = "^$*+?.()|{}[]\\";
   const regEx = new RegExp("[" + all.map((c) => (regExSpecial.includes(c) ? "\\" + c : c)).join("") + "]", "gim");
   return str.replace(regEx, "\\$&");
 };
 
-const extractNumberWithoutPrefix = (text) => {
+const extractNumberWithoutPrefix = (text) =>
+{
   const numberWithoutPrefix = text.replace(/^(-)?\d{3}/, "");
   return numberWithoutPrefix.length === 10 ? numberWithoutPrefix : null;
 };
 
-const cleanMessage = (text) => {
+const cleanMessage = (text) =>
+{
   // Remove all occurrences of @tag
   const cleanedText = text.replace(/@\w+/g, "");
 
   // Remove all occurrences of links (http and https)
   return cleanedText.replace(/(https?:\/\/[^\s]+)/g, "");
 };
 
-function extractTag(text) {
+const removeTag = (text) =>
+{
+  // Remove all occurrences of @tag
+  const cleanedText = text.replace(/@\w+/g, "").trim();
+  return cleanedText;
+};
+
+function extractTag(text)
+{
   const regex = /@(\w+)/;
   const match = regex.exec(text);
-  return match ? match[0] : null;
+  return match ? match[1] : null;
```

Comment:
This assumes that only the first tag is the assignee. Does it make sense to assume this or assign everybody? Or the last tag only? Please explain your reasoning.

Reply:
> This assumes that only the first tag is the assignee. Does it make sense to assume this or assign everybody? Or the last tag only? Please explain your reasoning.

This doesnt assume anything i moved from `[0]` to `[1]` because `[0]` added `@` to the result and `[1]` does not... i needed the tag without an `@` tag

Reply:
There is no multi-assign

Reply:
That doesn't answer my question. Anyways, we can merge assuming that only the first tag is the assignee.